#! /bin/sh -e

# DP: Mention the FAQ on the documentation index page.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- Doc/html/index.html.in~	2002-04-01 18:11:27.000000000 +0200
+++ Doc/html/index.html.in	2003-04-05 13:33:35.000000000 +0200
@@ -123,6 +123,24 @@
           </ul>
           </td>
         </tr>
+        <tr>
+          <td valign="baseline" class="left-column">
+          &nbsp;
+          <ul>
+            <li> <a href="../../python/FAQ.html" class="title"
+                >FAQ (local copy)</a>
+              <br>(python package must be installed)
+          </ul>
+          </td>
+          <td valign="baseline" class="right-column">
+          &nbsp;
+          <ul>
+            <li> <a href="http://www.python.org/cgi-bin/faqw.py" class="title"
+                >FAQ (online wizard)</a>
+              <br>(maybe more recent)
+          </ul>
+          </td>
+        </tr>
       </tbody>
     </table>
     <p>
